docs/src/test/scala/docs/http/scaladsl/HttpClientExampleSpec.scala

```diff
@@ -362,6 +366,10 @@ class HttpClientExampleSpec extends AnyWordSpec with Matchers with CompileOnlySp
       uri = "https://userservice.example/users",
       entity = HttpEntity(ContentTypes.`text/plain(UTF-8)`, "data")
     )
+
+    // or:
+    import akka.http.scaladsl.client.RequestBuilding.Post
+    Post("https://userservice.example/users", HttpEntity(ContentTypes.`text/plain(UTF-8)`, "data"))
```

Comment:
Have you tried this:

```suggestion
    Post("https://userservice.example/users", HttpEntity("data"))
```

or even

```suggestion
    Post("https://userservice.example/users", "data")
```